src/lumen/activity/football.cpp

```diff
@@ -3,6 +3,11 @@
 #include "lumen/hardware/display.hpp"
 
 namespace lumen::activity {
+namespace {
+
+constexpr auto tag = "activity/football";
```

Comment:
Was this necessary?🤣 I use it in my next PR, but this isn't used here. You can keep it though...